examples/teams/teams.json

```diff
@@ -50,10 +50,13 @@
     }
   },
   "Outputs": {
-    "Teams": {
-      "Description": " teams",
+    "TeamId": {
+      "Description": "TeamId",
       "Value": {
-        "Ref": "Teams"
+        "Fn::GetAtt": [
+          "Teams",
+          "TeamId"
+        ]
```

Comment:
Thanks! Also, what checks do we need to do to other resources to make sure it's not happening in other places? 

Looks like we've spotted this in a few different areas now - calls for a code review, or possibly a product of our planned manual QA.